### PR TITLE
Replace *.kproj with *.xproj

### PR DESCRIPTION
**Pull request by troydai**
@Eilon @muratg @danroth27

### CODE REVIEW

File: test/Microsoft.Framework.Runtime.Tests/FileGlobbing/ProjectFilesCollectionFacts.cs [hunk 1]
```diff
@@ -77,8 +77,8 @@ public void FilesPatternsAreSet()
 
             Assert.Equal(new string[] { "*.cs", "../*.cs" }, target.CompilePatternsGroup.IncludePatterns);
             Assert.Equal(new string[] { "signle.cs" }, target.CompilePatternsGroup.IncludeLiterals);
```
Comment (moozzyk) [inline test/Microsoft.Framework.Runtime.Tests/FileGlobbing/ProjectFilesCollectionFacts.cs]: nit: is it really supposed to be "signle.cs" ?

Comment (troydai) [inline test/Microsoft.Framework.Runtime.Tests/FileGlobbing/ProjectFilesCollectionFacts.cs]: Yeah, it's a typo. Same "signle.cs" is defined in previous set up.

Comment (troydai) [inline test/Microsoft.Framework.Runtime.Tests/FileGlobbing/ProjectFilesCollectionFacts.cs]: But won't be fixed here for the sake of limiting the scope of changes.

**Comment by muratg** [comment]
:shipit:

**Comment by Eilon** [comment]
:shipit: 

:exclamation: remember to push to release, then port to dev.

**Comment by troydai** [comment]
I'll merge to release.

**Pull request merged**